README.rst

```diff
@@ -0,0 +1,132 @@
+Introduction
+============
+
+This package provides a contact contenttype.
+
+.. contents:: Table of Contents
+
+
+Installation
+------------
+
+Add the package as dependency to your setup.py:
+
+.. code:: python
+
+  setup(...
+        install_requires=[
+          'ftw.contacts',
+        ])
+
+or to your buildout configuration:
+
+.. code:: ini
+
+  [instance]
+  eggs += ftw.contacts
+
+and rerun buildout.
+
+Install the Generic Setup profile.
+
+Motivation
+----------
+
+The ``ftw.contacts`` modernize the very old egov.contactdirectory and
+prepares the package for plone 5 integration.
+
+Quick overview of the functions
+-------------------------------
+
+- Contactdirectory with ajax search functionality
+- Contact
+- KML-Representation of Contact
+- vCard-Representation of Contact
+- ftw.zipexport integration if you install the [zipexport]-extra
+- ftw.simplelayout integration if you install the [simplelayout]-extra
+
+How it looks
+------------
+
+The ``ftw.contacts`package provides a nice contactdirectory.
+
+.. image:: https://raw.github.com/4teamwork/ftw.contacts/master/docs/contactdirectory.png
+
+with a contact-view.
+
+.. image:: https://raw.github.com/4teamwork/ftw.contacts/master/docs/contact.png
+
+How it works
+------------
+
+Contactfolder
+~~~~~~~~~~~~~
+
+After installing the ``ftw.contacts`` package you can add a ``Contactfolder``to your plonesite.
```

Comment:
`````Contactfolder``to``` is invalid, a space is missing: `````Contactfolder`` to```
there are some other places with the same problem, please take a look at the rendered readme :wink: